go-controller/pkg/ovn/obj_retry.go

```diff
@@ -1231,9 +1214,15 @@ func (oc *Controller) WatchResource(objectsToRetry *retryObjs) *factory.Handler
 		},
 		syncFunc) // adds all existing objects at startup
 
+	if err != nil {
+		klog.Errorf("WatchResource for resource %v. "+
```

Comment:
this can also just be part of the returned error message

Reply:
ack.